client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java

```diff
@@ -472,7 +474,7 @@ public String callServer(String api, Map<String, String> params, Map<String, Str
         Header header = NamingHttpUtil.builderHeader();
         
         String url;
-        if (curServer.startsWith(UtilAndComs.HTTPS) || curServer.startsWith(UtilAndComs.HTTP)) {
```

Comment:
Should UtilAndComs.HTTP and UtilAndComs.HTTPS be removed?

Reply:
It needs to be deleted and replaced by the constant in the common package

Reply:
> Should UtilAndComs.HTTP and UtilAndComs.HTTPS be removed?

I have deleted these two constants